Since we use nostdinc we don't rely on Linux definitions.

---
 buildrump.sh/src/sys/external/bsd/libnv/dist/nv_compat.h                |    4 
 buildrump.sh/src/sys/net/bpfjit.h                                       |    2 
 obj/hurd-debian/rumpkernel/buildrump.sh/src/tooldir/bin/nbhost-mkdep    |  267 ----------
 obj/hurd-debian/rumpkernel/buildrump.sh/src/tools/host-mkdep/host-mkdep |  267 ----------
 4 files changed, 3 insertions(+), 537 deletions(-)

--- a/buildrump.sh/src/sys/net/bpfjit.h
+++ b/buildrump.sh/src/sys/net/bpfjit.h
@@ -39,7 +39,7 @@
 
 #include <sys/types.h>
 
-#ifdef __linux
+#if 0 //def __linux
 #include <pcap-bpf.h>
 #else
 #include <net/bpf.h>
--- a/buildrump.sh/src/sys/external/bsd/libnv/dist/nv_compat.h
+++ b/buildrump.sh/src/sys/external/bsd/libnv/dist/nv_compat.h
@@ -57,13 +57,13 @@
 	    __attribute__((__format__ (__printf__, fmtarg, firstvararg)))
 #endif
 
-#ifdef __linux__
+#if 0 //def __linux__
 #include <endian.h>
 #else
 #include <sys/endian.h>
 #endif
 
-#ifdef __linux__
+#if 0 //def __linux__
 static inline uint32_t
 be32dec(const void *buf)
 {
